cocos/spine/lib/spine-define.ts

```diff
@@ -344,3 +356,6 @@ export function overrideSpineDefine (wasm): void {
     overrideProperty_Animation();
     overrideProperty_Skeleton();
 }
+
+// To avoid cycle dependence: spine-version.ts -> spine-version-4.2.ts -> spine-wasm-utils.ts -> spine-define.ts -> spine-version.ts
+(spine as any)._overrideSpineDefine = overrideSpineDefine;
```

Comment:
Since we have resolve the cycle dependency by adding spine-instantiate-xxx.ts, I think we don't need this hack anymore. Just import 'overrideSpineDefine' from 'spine-define' in spine-wasm-utils.ts.

Reply:
https://github.com/bofeng-song/engine/pull/19